modules/images/image-loading-optimization/class-ilo-url-metrics-group-collection.php

```diff
@@ -239,11 +232,27 @@ public function is_every_group_complete(): bool {
 	public function get_flattened_url_metrics(): array {
 		return array_merge(
 			...array_map(
-				static function ( ILO_URL_Metrics_Group $group ): array {
-					return iterator_to_array( $group );
-				},
-				$this->groups
+				'iterator_to_array',
+				iterator_to_array( $this )
```

Comment:
Maybe worth adding a comment here to explain because someone may easily think there's some duplication mistake here - which I know is not the case but may not be obvious.

Reply:
Good idea. It will look even more confusing once we update to PHP 8.1 with first-class callables:

```php
return array_merge(
	...array_map(
		iterator_to_array(...),
		iterator_to_array( $this )
	)
);
```

Added comment in 91c521eb2e65b3e20f3dcf8ca9bff523a6049fa1.